src/components/Bookings/Bookings.test.jsx

```diff
@@ -1,11 +1,11 @@
-import { describe, it, expect } from "vitest";
-import { render, screen } from "@testing-library/react";
-import Bookings from "./Bookings.jsx";
+import { describe, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import Bookings from './Bookings.jsx';
 
-describe("Bookings Component", () => {
-  it("renders a main element", () => {
+describe('Bookings Component', () => {
+  it('renders a main element', () => {
     render(<Bookings />);
-    const mainElement = screen.getByRole("main");
+    const mainElement = screen.getByRole('main');
     expect(mainElement).toBeInTheDocument();
   });
 });
```

Comment:
Good to see a test here! Have a think about what this is testing though, does it test just your componant?